src/compute/src/render/threshold.rs

```diff
@@ -68,7 +68,8 @@ where
 {
     match oks {
         SpecializedArrangement::RowUnit(inner) => {
-            let oks = threshold_arrangement(inner, name, logic);
+            let name = format!("{} [val: empty]", name);
+            let oks = threshold_arrangement(inner, name.as_str(), logic);
```

Comment:
Dito: `&name` here and below.

Reply:
Idem! And same for `dispatch_rearrange_iterative`.